ballerina/tests/01-connection-init-test.bal

```diff
@@ -108,18 +110,144 @@ function testWithOptionsWithErroneousSSL() returns error? {
                     password: "password"
                 },
                 cert: {
-                    path: turstStorePath,
+                    path: trustStorePath,
                     password: "password"
                 }
             }
      };
-    Client oracledbClient = check new(
+    Client|error oracledbClient = new(
         host = HOST,
         user = USER,
         password = PASSWORD,
         port = PORT,
         database = DATABASE,
         options = options
     );
-    test:assertEquals(oracledbClient.close(), (), "Client Error");
+    test:assertTrue(oracledbClient is error);
+    if oracledbClient is sql:ApplicationError {
+        test:assertTrue(oracledbClient.message().startsWith("Error in SQL connector configuration: Failed to initialize pool: " +
+        "IO Error: closing inbound before receiving peer's close_notify"));
+    } else {
+        test:assertFail("Error ApplicatonError expected");
+    }
+}
+
+// with all params and options with Erroneous SSL with correct port
+@test:Config {
+    groups:["connection"]
+}
+function testWithOptionsWithErroneousSSLCorrctPort() returns error? {
+     Options options = {
+            ssl: {
+                key: {
+                    path: clientStorePath,
+                    password: "password"
+                },
+                cert: {
+                    path: trustStorePath,
+                    password: "password"
+                }
+            }
+     };
+    Client|error oracledbClient = new(
+        host = HOST,
+        user = USER,
+        password = PASSWORD,
+        port = SSLPORT,
+        database = DATABASE,
+        options = options
+    );
+    test:assertTrue(oracledbClient is error);
+    if oracledbClient is sql:ApplicationError {
+        test:assertTrue(oracledbClient.message().startsWith("Error in SQL connector configuration: Failed to initialize pool: " +
+        "IO Error: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException"));
```

Comment:
Shall we improve the error message without passing java exception

Reply:
An issue was created to track and resolve printing java class exceptions
https://github.com/ballerina-platform/ballerina-standard-library/issues/1775